contrib/util/ccda_import/batchImport.php

```diff
@@ -0,0 +1,188 @@
+$site_default = $_SESSION['site_id'] ?? 'default';
+$isDev_default = 'false';
+$enableMoves_default = 'false';
+$dedup_default = 'true';
+
+// Set form value variables.
+$sourcePath_val = isset($_POST['sourcePath']) ? trim($_POST['sourcePath']) : $sourcePath_default;
+$openemrPath_val = isset($_POST['openemrPath']) ? trim($_POST['openemrPath']) : $openemrPath_default;
+$site_val = isset($_POST['site']) ? trim($_POST['site']) : $site_default;
+$isDev_val = isset($_POST['isDev']) ? trim($_POST['isDev']) : $isDev_default;
+$enableMoves_val = isset($_POST['enableMoves']) ? trim($_POST['enableMoves']) : $enableMoves_default;
+$dedup_val = isset($_POST['dedup']) ? trim($_POST['dedup']) : $dedup_default;
+
+ob_implicit_flush(true);
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8">
+    <?php Header::setupHeader(); ?>
+    <title><?php echo xlt('CCDA Batch Import'); ?></title>
+    <script>
+        function scrollToBottom() {
+            setTimeout(() => {
+                window.scrollTo({top: document.body.scrollHeight, behavior: 'smooth'});
+            }, 0);
+        }
+    </script>
+</head>
+<body>
+    <div class="container mt-5">
+        <h1 class="mb-1"><?php echo xlt('Batch Import CCDAs'); ?></h1>
+        <div class="col-12">
+            <label><?php echo xlt('This tool allows you to batch import CCDAs into OpenEMR.'); ?></label><br />
+            <label><?php echo xlt('Please ensure that the source path contains the CCDAs to be imported.'); ?></label><br />
+            <label><?php echo xlt('The OpenEMR path is where OpenEMR is installed.'); ?></label><br />
+            <p>
+                <?php
+                echo "<h5>Usage: php import_ccda.php [OPTIONS]</h5>";
+                echo "Options:<br />";
+                echo "  --authName      Required if isDev=false. userAuth so Documents can be saved/moved.<br />";
+                echo "  --sourcePath     Required. Path to the directory containing CCDA files to import.<br />";
+                echo "  --site           Required. OpenEMR site ID.<br />";
+                echo "  --openemrPath    Required. Path to OpenEMR web root.<br />";
+                echo "  --isDev          Optional. Set to 'true' for development mode, 'false' for production. Default: true.<br />";
+                echo "  --enableMoves    Optional. Set to 'true' to move processed files, 'false' to disable. Default: false.<br />";
+                echo "  --dedup          Optional. Set to 'true' to enable duplicate checking, 'false' to disable. Default: false.<br />";
+                echo "  --help           Show this help message.<br />";
+                ?>
+            </p><br />
+        </div>
+        <div class="col-12">
+            <form id="importForm" method="post" onload="scrollToBottom()">
+                <div class="form-group">
+                    <label for="sourcePath">Source Path:</label>
```

Comment:
Translate the labels in this script (ensure do not include the colons on the labels)